lib/TvmClient.js

```diff
@@ -325,6 +354,7 @@ class TvmClient {
 TvmClient.AzureBlobEndpoint = 'azure/blob'
 TvmClient.AwsS3Endpoint = 'aws/s3'
 TvmClient.AzureCosmosEndpoint = 'azure/cosmos'
+TvmClient.CredsCacheVar = 'credsCache'
```

Comment:
I would change this to `TvmClient.memoryCache = {}` and access the cache with `TvmClient.memoryCache[cacheKey]` instead of `TvmClient[TvmClient.CredsCacheVar][cacheKey]`

Reply:
instance var initialized in constructor? Isn't that what we are trying to solve?

Reply:
@Himavanth right forgot about cross instance invocations, thanks! 
In that case I would still suggest the minor change to `TvmClient.MemoryCache` and `TvmClient.MemoryCache[cacheKey]`as it is more direct `TvmClient[TvmClient.CredsCacheVar][cacheKey]`